DXRVanilla/DeusEx/Classes/Player.uc

```diff
@@ -2200,6 +2200,63 @@ exec function ActivateBelt(int objectNum)
     }
 }
 
+function CompleteBingoGoal(PlayerDataItem data, int x, int y)
+{
+    local string event;
+    local int progress, max;
+
+    data.GetBingoSpot(x, y, event,, progress, max);
+    while (progress < max) {
+        class'DXREventsBase'.static.MarkBingo(event);
+        progress++;
+    }
+}
+
+exec function Bingo(int line)
```

Comment:
we actually have a preproc variable for this, which makes the goals completed when clicked

```
    if(#defined(bingocheat)) {
        class'DXREvents'.static.MarkBingo(bt.event);
        bt.progress++;
    }
```

I guess this is fine too

Reply:
I'd prefer to always be able to click the goals without completing them.